### PR TITLE
Add local level model

### PR DESCRIPTION
**Pull request by guilhermebodin**
For some reason, Optim is having trouble in the multivariate case. We must see this before merging

### CODE REVIEW

**Comment by raphaelsaavedra** [comment]
Change all `pinv.` calls to `pinv` to avoid problems in multivariate case

File: src/models.jl [hunk 1]
```diff
@@ -89,3 +89,34 @@ function structuralmodel(y::VecOrMat{Typ}, s::Int; X::VecOrMat{Typ} = Matrix{Flo
...
+    y = y[:, :]
+    n, p = size(y)
+
+    @info("Creating local level model with $p endogenous variables.")
+
+    m = p
+    r = p
+
+    # Observation equation
+    Z = kron(Matrix{Float64}(I, p, p), [1])
```
Comment (raphaelsaavedra) [inline src/models.jl]: These krons do nothing since the result of kron(I, 1) is I, you could simply do `Z = Matrix{Float64}(I, p, p)`

**Comment by raphaelsaavedra** [comment]
Also it's important to have at least one multivariate test. I suggest the following:

```
y = [ones(20) collect(1:20)]
model = structuralmodel(y, 2)
ss = statespace(model)
sim  = simulate(ss, 10, 1000)

@test mean(sim, dims = 3)[1, :] ≈ ones(10) rtol = 1e-3
@test mean(sim, dims = 3)[2, :] ≈ collect(21:30) rtol = 1e-3
```

Can be changed to linear trend once `lineartrendmodel` it is implemented since the seasonality is useless in this case.

File: src/models.jl (hunk 1 above)
Comment (guilhermebodin) [reply src/models.jl]: done!

**Pull request merged**